client/src/components/DatasetInformation/DatasetError.vue

```diff
@@ -62,11 +62,8 @@
                         >{{ resultMessage[0] }}</b-alert
                     >
                     <div v-if="showForm" id="fieldsAndButton">
-                        <FormElement
-                            v-if="!jobDetails.user_email"
-                            id="dataset-error-email"
-                            v-model="email"
-                            title="Please provide your email:" />
+                        <span class="mr-2 font-weight-bold">{{ emailTitle }}</span>
```

Comment:
Here you can just write `<span class="mr-2 font-weight-bold">{{ emailTitle | l }}</span>` and avoid the localization import below.

Reply:
@guerler Yes, that's better. Change made.

Reply:
Thanks nice option is to us `v-localize`. Thank you.